src/simtools/schemas/model_parameters/dsum_shaping_renormalize.schema.yml

```diff
@@ -18,7 +18,6 @@ instrument:
   type:
     - MSTN
     - MSTS
-    - SSTS
```

Comment:
Why did you remove only SSTs if it is relevant only for FlashCams?

Reply:
Same, see above.